app/client/components/card.tsx

```diff
@@ -0,0 +1,50 @@
2LTEuMi0uNmMtMS40IDAtMi4xIDEuNC0yLjEgMi45IDAgMSAuNSAyIDEuNSAyIC43IDAgMS4xLS41IDEuMS0uNWwtLjEuNGgxLjJsLjktNS45aC0xLjF6bS0uNSAzLjJjMCAuNy0uMyAxLjUtMSAxLjUtLjQgMC0uNi0uNC0uNi0xIDAtMSAuNC0xLjYgMS0xLjYuNC4xLjYuNC42IDEuMXpNOS43IDIwLjlsLjctNC40LjEgNC40aC44bDEuNi00LjQtLjcgNC40aDEuM2wxLTUuOWgtMmwtMS4yIDMuNlYxNUg5LjRsLTEgNS45aDEuM3ptMTguOSAwYy40LTIgLjUtMy42IDEuNS0zLjMuMi0uOC41LTEuNS43LTEuOCAwIDAtLjEtLjEtLjQtLjEtLjYgMC0xLjMgMS4xLTEuMyAxLjFsLjEtLjdoLTEuMWwtLjggNC44aDEuM3pNMzYgMTZjLS44IDAtMS41LjMtMS41LjNsLS4yIDEuMXMuNS0uMiAxLjMtLjJjLjUgMCAuOC4xLjguNHYuM2gtLjVjLTEuMiAwLTIuMi41LTIuMiAxLjggMCAxLjEuNyAxLjMgMS4yIDEuMy45IDAgMS4yLS41IDEuMi0uNnYuNWgxLjFsLjUtMy40YzAtMS40LTEuMy0xLjUtMS43LTEuNXptLjIgMi44YzAgLjItLjEgMS4yLS44IDEuMi0uNCAwLS41LS4zLS41LS41IDAtLjMuMi0uNyAxLjEtLjdoLjJ6bTIuNyAyLjFjLjItMS41LjYtMy42IDEuNS0zLjMuMi0uOCAwLS44LS4zLS44aC0uN2wuMS0uN2gtMS4xbC0uOCA0LjhoMS4zeiIvPjxwYXRoIGZpbGw9IiNmZmYiIGQ9Ik0yMy40IDIwLjVsLjItMWgtLjNjLS40IDAtLjUtLjItLjQtLjRsLjMtMi4xaC42bC4yLTEuMmgtLjZsLjEtLjdoLTEuMnMtLjcgNC0uNyA0LjVjMCAuNy40IDEgMSAxIC4zIDAgLjctLjEuOC0uMXptLjQtMmMwIDEuNyAxLjEgMi4yIDIuMSAyLjIuOSAwIDEuMy0uMiAxLjMtLjJsLjItMS4ycy0uNy4zLTEuMy4zYy0xLjMgMC0xLjEtMS0xLjEtMWgyLjVzLjItLjguMi0xLjFjMC0uOC0uNC0xLjgtMS44LTEuOC0xLjItLjEtMi4xIDEuMy0yLjEgMi44em0yLjItMS44Yy43IDAgLjYuOC42LjhoLTEuNHMuMS0uOC44LS44em03LjcgMy44bC4yLTEuM3MtLjYuMy0xIC4zYy0uOSAwLTEuMi0uNy0xLjItMS40IDAtMS41LjgtMi4zIDEuNi0yLjMuNiAwIDEuMS40IDEuMS40bC4yLTEuM3MtLjctLjMtMS40LS4zYy0xLjQgMC0yLjggMS4yLTIuOCAzLjUgMCAxLjUuNyAyLjUgMi4yIDIuNS41IDAgMS4xLS4xIDEuMS0uMXptLTE3LTQuOWMtLjggMC0xLjUuMy0xLjUuM0wxNSAxN3MuNS0uMiAxLjMtLjJjLjUgMCAuOC4xLjguNHYuM2gtLjVjLTEuMSAwLTIuMi41LTIuMiAxLjggMCAxLjEuNyAxLjMgMS4yIDEuMy45IDAgMS4yLS42IDEuMy0uNnYuNUgxOGwuNS0zLjRjLS4yLTEuNC0xLjQtMS41LTEuOC0xLjV6bS4yIDIuOGMwIC4yLS4xIDEuMi0uOSAxLjItLjQgMC0uNS0uMy0uNS0uNSAwLS4zLjItLjcgMS4xLS43aC4zem0yLjYgMi4yYy4zIDAgMS45LjEgMS45LTEuNiAwLTEuNS0xLjUtMS4yLTEuNS0xLjggMC0uMy4yLS40LjctLjQuMiAwIC44LjEuOC4xbC4yLTEuMXMtLjQtLjEtMS4xLS4xYy0uOSAwLTEuOC40LTEuOCAxLjYgMCAxLjQgMS41IDEuMyAxLjUgMS45IDAgLjQtLjQuNC0uOC40LS42IDAtMS4xLS4yLTEuMS0uMmwtLjIgMS4xYzAtLjEuNC4xIDEuNC4xem0yNC40LTUuOWwtLjMgMS42cy0uNS0uNi0xLjItLjZjLTEuMSAwLTIuMSAxLjQtMi4xIDIuOSAwIDEgLjUgMiAxLjUgMiAuNyAwIDEuMS0uNSAxLjEtLjVsLjEuNGgxLjJsLjktNS45LTEuMi4xem0tLjYgMy4yYzAgLjYtLjMgMS41LTEgMS41LS40IDAtLjYtLjQtLjYtMSAwLTEgLjQtMS42IDEtMS42LjQuMS42LjQuNiAxLjF6TTEwIDIwLjVsLjctNC40LjEgNC40aC44bDEuNi00LjQtLjcgNC40aDEuM2wxLTUuOWgtMS45bC0xLjIgMy42LS4xLTMuNkg5LjhsLTEgNS45SDEwem0xOC45IDBjLjQtMiAuNC0zLjcgMS4zLTMuNC4xLS44LjMtMS4xLjUtMS40aC0uMmMtLjYgMC0xIC44LTEgLjhsLjEtLjdoLTEuMWwtLjggNC44aDEuMnYtLjF6bTcuNy00LjljLS44IDAtMS41LjMtMS41LjNsLS4yIDEuMXMuNS0uMiAxLjMtLjJjLjUgMCAuOC4xLjguNHYuM2gtLjVjLTEuMSAwLTIuMi41LTIuMiAxLjggMCAxLjEuNyAxLjMgMS4yIDEuMy45IDAgMS4yLS42IDEuMy0uNnYuNWgxLjFsLjUtMy40Yy0uMS0xLjQtMS40LTEuNS0xLjgtMS41em0uMiAyLjhjMCAuMi0uMSAxLjItLjkgMS4yLS40IDAtLjUtLjMtLjUtLjUgMC0uMy4yLS43IDEuMS0uN2guM3ptMi40IDIuMWMuNC0yIC40LTMuNyAxLjMtMy40LjEtLjguMy0xLjEuNS0xLjRoLS4yYy0uNiAwLTEgLjgtMSAuOGwuMS0uN2gtMS4xbC0uOCA0LjdoMS4yeiIvPjwvc3ZnPg==)";
```

Comment:
I'm not a fan of these big inline svgs, I'm not sure what the best way of doing this is though.